src/services/positionService.ts

```diff
@@ -755,70 +752,59 @@ export default class PositionService {
       (position.from.underlyingTokens.length && position.remainingLiquidity.gt(BigNumber.from(0))) ||
       (position.to.underlyingTokens.length && position.toWithdraw.gt(BigNumber.from(0)));
 
-    if (!useProtocolToken && !hasYield) {
-      const hubInstance = await this.contractService.getHubInstance(position.version);
-
-      return hubInstance.terminate(
-        position.positionId,
-        this.walletService.getAccount(),
-        this.walletService.getAccount()
-      );
-    }
-
     const companionHasPermission = await this.companionHasPermission(position, DCAPermission.TERMINATE);
 
-    if (companionHasPermission) {
-      return this.meanApiService.terminateUsingOtherTokens(
```

Comment:
Don't you want to delete code from `this.meanApiService. terminateUsingOtherTokens `?